<launch>
  <arg name="gt_topic" default="/tesse/odom"/>
  <arg name="output_dir" default="$(find kimera_vio_ros)/output_logs/"/>

  <node name="gt_logger_node" pkg="kimera_vio_ros" type="gt_logger_node.py"
        output="screen">
    <param name="gt_topic"   value="$(arg gt_topic)"/>
    <param name="output_dir" value="$(arg output_dir)"/>
  </node>
</launch>
